examples/exotica_examples/resources/configs/ik_with_jnt_smoothing.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" ?>
+<IKSolverDemoConfig>
+
+  <IKsolver Name="MySolver">
+    <MaxIterations>1</MaxIterations>
+    <MaxStep>0.1</MaxStep>
+    <Tolerance>1e-5</Tolerance>
+    <Alpha>1.0</Alpha>
+    <C>1e-3</C>
+  </IKsolver>
+
+  <UnconstrainedEndPoseProblem Name="MyProblem">
+
+    <PlanningScene>
+      <Scene>
+        <JointGroup>arm</JointGroup>
+        <URDF>{exotica_examples}/resources/robots/lwr_simplified.urdf</URDF>
+        <SRDF>{exotica_examples}/resources/robots/lwr_simplified.srdf</SRDF>
+      </Scene>
+      
+    </PlanningScene>
+    
+    <Maps>
+      
+      <EffFrame Name="Position">
+        <EndEffector>
+          <Frame Link="lwr_arm_6_link"
+		 LinkOffset="0 0 0 0.7071 0 0.7071 0"/>
+        </EndEffector>
+      </EffFrame>
+
+      <JointVelocityBackwardDifference Name="JointVel">
+	<StartState>1.147249698638916 -0.22851833701133728 0.5324112772941589 1.2311428785324097 -0.19176392257213593 -0.1434374898672104 0</StartState>
```

Comment:
XML Formatting strikes again

Reply:
This is strange, in my local repo the formatting is indented as expected.

Reply:
Tabs vs spaces.
Always use spaces (and that's how the war began).